beetsplug/export.py

```diff
@@ -52,6 +57,24 @@ def __init__(self):
                     'sort_keys': True
                 }
             },
+            'csv': {
+                # csv module formatting options
+                'formatting': {
+                    'ensure_ascii': False,
+                    'indent': 0,
+                    'separators': (','),
+                    'sort_keys': True
+                }
+            },
+            'xml': {
+                # xml module formatting options
+                'formatting': {
+                    'ensure_ascii': False,
+                    'indent': 4,
+                    'separators': ('>'),
+                    'sort_keys': True
+                }
```

Comment:
I don't think any of these config options are actually used in the CSV or XML emitters.

Reply:
I was just trying to follow the same format, but you are right they are not being used. I have updated them to now be format specific to json, csv and xml and updated the csv and xml outputs to use them...
```
'json': {
                # json module formatting options
                'formatting': {
                    'ensure_ascii': False,
                    'indent': 4,
                    'separators': (',', ': '),
                    'sort_keys': True
                }
            },
            'csv': {
                # csv module formatting options
                'formatting': {
                    'delimiter': ',', # column seperator
                    'dialect': 'excel', # the name of the dialect to use
                    'quotechar':'|'
                }
            },
            'xml': {
                # xml module formatting options
                'formatting': {
                    'encoding': 'UTF-8', # the output encoding
                    'xml_declaration':'True', # controls if an XML declaration should be added to the file
                    'default_namespace': 'None', # sets the default XML namespace (for “xmlns”)
                    'method': 'xml', # either "xml", "html" or "text" (default is "xml")
                    'short_empty_elements': 'True' # controls the formatting of elements that contain no content.
                }
```